glean_parser/templates/rust_server.jinja2

```diff
@@ -0,0 +1,316 @@
+{# The final Rust code is autogenerated, but this template is not. Please file bugs if there is a problem! #}
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// AUTOGENERATED BY glean_parser v{{ parser_version }}. DO NOT EDIT.
+
+// Required imports
+use chrono::Utc;
+use serde::{Deserialize, Serialize};
+use serde_json;
+use std::collections::HashMap;
```

Comment:
Couple of places use the full path `std::collections::HashMap`, others use the shortened enabled by this import, `HashMap`.
Feels like we should keep this consistent.

Reply:
Good call, not sure why clippy (and I) missed that.